SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -277,7 +280,7 @@
 
 			volumeNum = gui.numberBox.new(w, Rect(0, 0, 28, 18))
 				.font_(font)
-				.value_(0.0)
+				.value_(currentVolume.clip(-90, 6))
```

Comment:
Correct me if I'm wrong, but I don't think you need to clip here. The Volume object itself clips, and can be set to have different ranges than -90 or 6, so not good to enforce the defaults in the GUI, esp as they may contradict the actual state.